.gitignore

```diff
@@ -3,4 +3,4 @@ composer.lock
 composer.phar
 phpcs.xml
 phpstan.neon
-vendor/
+vendor/
```

Comment:
we're missing a line feed

If you're using Jetbrains IDE, it can be automatically handled by `Editor -> General -> Other`

![image](https://user-images.githubusercontent.com/327717/51794287-b7f8e600-21cf-11e9-9b8b-3b61fc391131.png)